diff --git a/tests/test-string-output-visitor.c b/tests/test-string-output-visitor.c
index 101fb27dd1..fd5e67be64 100644
--- a/tests/test-string-output-visitor.c
+++ b/tests/test-string-output-visitor.c
@@ -251 +251 @@ int main(int argc, char **argv)
-    output_visitor_test_add("/string-visitor/output/int",
+    output_visitor_test_add("/string-visitor/output/int-human",
@@ -255 +255 @@ int main(int argc, char **argv)
-    output_visitor_test_add("/string-visitor/output/bool",
+    output_visitor_test_add("/string-visitor/output/bool-human",
@@ -259 +259 @@ int main(int argc, char **argv)
-    output_visitor_test_add("/string-visitor/output/number",
+    output_visitor_test_add("/string-visitor/output/number-human",
@@ -263 +263 @@ int main(int argc, char **argv)
-    output_visitor_test_add("/string-visitor/output/string",
+    output_visitor_test_add("/string-visitor/output/string-human",
@@ -268 +268 @@ int main(int argc, char **argv)
-    output_visitor_test_add("/string-visitor/output/no-string",
+    output_visitor_test_add("/string-visitor/output/no-string-human",
@@ -273 +273 @@ int main(int argc, char **argv)
-    output_visitor_test_add("/string-visitor/output/enum",
+    output_visitor_test_add("/string-visitor/output/enum-human",
@@ -278 +278 @@ int main(int argc, char **argv)
-    output_visitor_test_add("/string-visitor/output/enum-errors",
+    output_visitor_test_add("/string-visitor/output/enum-errors-human",
@@ -283 +283 @@ int main(int argc, char **argv)
-    output_visitor_test_add("/string-visitor/output/intList",
+    output_visitor_test_add("/string-visitor/output/intList-human",
